whylog/log_reader/searchers.py

```diff
@@ -88,6 +88,7 @@ def search(self, investigation_step):
         clues = defaultdict(list)
         offset = self._deduce_offset(investigation_step)
         for line, actual_offset in self._reverse_from_offset(offset):
+            #TODO: passing to get_clues invocation line source
```

Comment:
I do not understand the comment. Do you mean that I should to pass also `line_source` object into `get_clues` function?